dusty/commands/repos.py

```diff
@@ -21,10 +21,19 @@ def list_repos():
                        repo.override_path if repo.is_overridden else ''])
     log_to_client(table.get_string(sortby='Short Name'))
 
+def nfs_path_exists(path):
```

Comment:
This function seems insane if you don't know why it's here, docstring would be great